src/components/common/Ovals/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { Image } from 'react-native';
+import styles from './styles';
+
+const Ovals = () => <Image source={require('assets/images/ovals.png')} style={styles.image} />;
```

Comment:
It's not a good practice to have this URL here, you can import the asset and the pass as source